include/accessor.h

```diff
@@ -311,13 +316,20 @@ class accessor<DataT, Dims, Mode, target::device> : public detail::accessor_base
 	DataT* m_device_ptr = nullptr;
 	CELERITY_DETAIL_NO_UNIQUE_ADDRESS id<Dims> m_backing_buffer_offset;
 	CELERITY_DETAIL_NO_UNIQUE_ADDRESS range<Dims> m_backing_buffer_range = detail::zero_range;
+#if CELERITY_ACCESSOR_BOUNDARY_CHECK
+	detail::buffer_id m_accessed_buffer_id; // = nullptr;
```

Comment:
This doesn't seem to get used anywhere, remnant of an earlier implementation?

Reply:
I can't recall ATM which was the specific use case, but I recall in a few situations wondering why do accessors don't know their buffer's id, and yes, also most likely earlier impl.